UPGRADING

```diff
@@ -623,6 +635,12 @@ PHP 8.5 UPGRADE NOTES
 7. New Classes and Interfaces
 ========================================
 
+- Core:
+  . DelayedTargetValidation is an attribute that, when added, delays any errors
+    from *other* internal attributes about being applied to invalid targets from
+    compile-time to runtime.
+    RFC: https://wiki.php.net/rfc/delayedtargetvalidation_attribute
```

Comment:
Not sure if we ever added attributes here. Makes sense, but I would just mention the class name + RFC and not the full explanation. Can you just commit the related change for NoDiscard to master?